powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/OperationApproveRequest.java

```diff
@@ -32,6 +34,7 @@
     private Long applicationId;
     private String data;
     private SignatureType signatureType;
+    private Map<String, String> additionalData;
```

Comment:
Small suggestion: perhaps we can make the map `final` and initialize it with an empty `LinkedHashMap`, this way we can avoid potential nullability issues?